nevergrad/functions/images/core.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from math import sqrt, tan, pi
+import numpy as np
+import nevergrad as ng
+import PIL
+import PIL.Image
+import os
+from nevergrad.common.typetools import ArrayLike
+from .. import base
+
+
+class Image(base.ExperimentFunction):
+    def __init__(self, problem_type: str = "recovering", index_pb: int = 0) -> None:
+        """
+        problem_type: the type of problem we are working on.
+           recovering: we directly try to recover the target image.
+        index_pb: the index of the problem, inside the problem type.
+           For example, if problem_type is "recovering" and index_pb == 0,
+           we try to recover the face of O. Teytaud.
+        """
+
+        # Storing high level information.
+        self.domain_shape = (256, 256, 3)
+        self.problem_type = problem_type
+        self.index_pb = index_pb
+
+        # Storing data necessary for the problem at hand.
+        assert problem_type == "recovering"  # For the moment we have only this one.
+        assert index_pb == 0  # For the moment only 1 target.
+        path = os.path.dirname(__file__) + "/headrgb_olivier.png"
```

Comment:
Prefer `Path` over `os.path`. I think the following would work:
```python
from pathlib import Path
path = Path(__file__).with_name("headrgb_olivier.png")
```

Reply:
Done.